src/EnergyPlus/WeatherManager.cc

```diff
@@ -2557,121 +2369,74 @@ namespace WeatherManager {
-                                                                                   "Diffuse Radiation",
-                                                                                   "WeatherFile",
-                                                                                   "Severe",
-                                                                                   ">= 0",
-                                                                                   (DiffuseRad >= 0.0),
-                                                                                   {},
-                                                                                   state.dataEnvrn->WeatherFileLocationTitle);
-                    if (WindDir < 999.0)
-                        state.dataInputProcessing->inputProcessor->rangeCheck(state,
-                                                                              ErrorsFound,
-                                                                              "Wind Direction",
-                                                                              "WeatherFile",
-                                                                              "Severe",
-                                                                              ">=0",
-                                                                              (WindDir >= 0.0),
-                                                                              "<=360",
-                                                                              (WindDir <= 360.0),
-                                                                              format("{:.0R}", WindDir),
-                                                                              state.dataEnvrn->WeatherFileLocationTitle);
-                    if (WindSpeed < 999.0)
-                        state.dataInputProcessing->inputProcessor->rangeCheck(state,
-                                                                              ErrorsFound,
-                                                                              "Wind Speed",
-                                                                              "WeatherFile",
-                                                                              "Severe",
-                                                                              ">=0",
-                                                                              (WindSpeed >= 0.0),
-                                                                              "<=40",
-                                                                              (WindSpeed <= 40.0),
-                                                                              format("{:.2R}", WindSpeed),
-                                                                              state.dataEnvrn->WeatherFileLocationTitle);
+                    if (DryBulb < 99.9 && (DryBulb < -90.0 || DryBulb > 70.0)) {
```

Comment:
This idiom where we pass a bunch of strings to a function (not to mention call the function to begin with) for the rare case of errors has to stop.